en/assembly/mount_gps_compass.md

```diff
@@ -7,13 +7,25 @@ The diagram below shows the heading marker on the Pixhawk 4 and compass.
 
 ## Compass Orientation
 
-The compass can also be mounted in any other _supported_ orientation, where the supported orientations can be seen in [CAL_MAGn_ROT](../advanced_config/parameter_reference.md#CAL_MAG1_ROT), and have the same meaning as used for [orienting the flight controller](../config/flight_controller_orientation.md#calculating-orientation).
+The compass can also be mounted in any _supported_ orientation, where the supported orientations can be seen in [CAL_MAGn_ROT](../advanced_config/parameter_reference.md#CAL_MAG1_ROT), and uses the same convention for [orienting the flight controller](../config/flight_controller_orientation.md#orientation-definition).
```

Comment:
"other" was deliberate - we want them to know that if they mount as recommended in the very first para of the doc they can ignore the rest. 

This text is OK too, but what that makes me think is that perhaps we should "spell this out" in the first paragraph - i'e if you do this don't bother reading the rest.

Reply:
I'll just do it :-)